src/js/components/homepage/Homepage.jsx

```diff
@@ -19,6 +19,7 @@ const Homepage = () => (
     <PageWrapper
         pageName="Homepage"
         classNames="usa-da-home-page"
+        title="Government Spending Open Data | USAspending"
```

Comment:
other instances of PageWrapper title seem to match exactly whatever the pageName prop string is